docs/source/release-notes/2021-1-2-Eos.md

```diff
@@ -0,0 +1,73 @@
+# 2021.1.2 Eos Release Notes
+
+This version of Vircadia is an update to 2021.1.1. It is compatible with the 2020.1.0 Eos protocol. This version was released on May 27th, 2021.
+
+## Native Interface & Domain Server (Codename Athena)
+
+The numbers at the end of each item are the PR numbers in the Vircadia [repo](https://github.com/vircadia/vircadia).
+
+### Enhancements
+
+- [**osx**][**windows**] Hash Windows and Mac machine fingerprints. [#1126](https://github.com/vircadia/vircadia/pull/1126)
+- [**domain-server**] The domain server should see Packet ID=1 as a (broken) attempt to connect to the domain server [#692](https://github.com/vircadia/vircadia/pull/692)
+-  Create app: Copy-Paste Position and Rotation [#1213](https://github.com/vircadia/vircadia/pull/1213)
+-  Add Doxygen tool for compiling C++ documentation [#1212](https://github.com/vircadia/vircadia/pull/1212)
+-  Include Vircadia Web app as a Git submodule [#1202](https://github.com/vircadia/vircadia/pull/1202)
+-  ReadyPlayerMe blendshapes support [#1183](https://github.com/vircadia/vircadia/pull/1183)
+-  Update to Qt 5.15.2 [#938](https://github.com/vircadia/vircadia/pull/938)
+- [**osx**] Revise OSX DMGs [#1162](https://github.com/vircadia/vircadia/pull/1162)
+-  Implement VIRCADIA_MEMORY_DEBUGGING for Windows [#1171](https://github.com/vircadia/vircadia/pull/1171)
+
+### Bug Fixes
+
+- [**linux**] Link to pthread last [#1188](https://github.com/vircadia/vircadia/pull/1188)
+-  Update Snapshot.cpp [#1193](https://github.com/vircadia/vircadia/pull/1193)
+- [**linux**] Add patch for building Qt 5.15 on gcc11 [#1218](https://github.com/vircadia/vircadia/pull/1218)
+-  Fix mismatched if in CMake warning [#1173](https://github.com/vircadia/vircadia/pull/1173)
+- [**android**] fix PR autobuild for ubuntu-18.04, android with correct cmake version [#1225](https://github.com/vircadia/vircadia/pull/1225)
+-  Fix GitHub Actions merge-to-master MacOS build [#1199](https://github.com/vircadia/vircadia/pull/1199)
+-  Fix settings (QML) spin box stepping [#1161](https://github.com/vircadia/vircadia/pull/1161)
+- [**renderer**] Fix glTF mesh colors [#1181](https://github.com/vircadia/vircadia/pull/1181)
+-  Fix LOD manager defaults + settings not being saved correctly. [#1129](https://github.com/vircadia/vircadia/pull/1129)
+-  Fix minitablet goto [#1182](https://github.com/vircadia/vircadia/pull/1182)
+
+### Docs
+
+- [**docs**] Fix missed JSDoc tag change [#1217](https://github.com/vircadia/vircadia/pull/1217)
+- [**docs**] Fix typo in Doxygen ReadMe [#1220](https://github.com/vircadia/vircadia/pull/1220)
```

Comment:
Could just append the PR number to the previous item, "Add Doxygen tool for compiling C++ documentation".